src/main/java/frc/robot/commands/LiftUpCommand.java

```diff
@@ -37,5 +39,6 @@ public boolean isFinished() {
     public void end(boolean interrupted) {
         lift.enableRatchets();
         lift.disable();
+        drawbridge.up();
```

Comment:
I have concerns about this. What if you use this button to back off the chain slightly after scoring in trap and the drawbridges try to go up?